python/tk_framework_desktopserver/certificates.py

```diff
@@ -119,8 +120,8 @@ def _check_call(self, ctx, cmd):
         else:
             p = subprocess.Popen(cmd, stderr=subprocess.STDOUT, stdout=subprocess.PIPE, shell=True)
         stdout, _ = p.communicate()
+        self._logger.debug("Stdout:\n%s" % stdout)
```

Comment:
Always log the output, we never know what kind of silent failures our clients might encounter.